src/visGeometry/index.ts

```diff
@@ -1166,57 +1166,43 @@ class VisGeometry {
             // get geom for lookupKey,
             // will only load each geometry once, so may return nothing
             // if the same geometry is assigned to more than one agent
-            this.geometryStore
-                .mapKeyToGeom(Number(id), entry.geometry)
-                .then((newGeometryLoaded) => {
-                    if (!newGeometryLoaded) {
-                        // no new geometry to load
-                        return;
-                    }
-                    // will only have a returned displayType if it changed.
-                    const {
-                        displayType: returnedDisplayType,
-                        geometry,
-                        errorMessage,
-                    } = newGeometryLoaded;
-                    const newDisplayType = returnedDisplayType || displayType;
-                    this.onNewRuntimeGeometryType(
-                        lookupKey,
-                        newDisplayType,
-                        geometry
-                    );
-                    // handle additional async update to LOD for pdbs
-                    if (
-                        newDisplayType === GeometryDisplayType.PDB &&
-                        geometry
-                    ) {
-                        const pdbModel = geometry as PDBModel;
-                        return pdbModel.generateLOD().then(() => {
-                            this.logger.info(
-                                "Finished loading pdb LODs: ",
-                                lookupKey
-                            );
-                            this.onNewRuntimeGeometryType(
-                                lookupKey,
-                                newDisplayType,
-                                geometry
-                            );
-                        });
-                    }
-                    // if returned with a resolve, but has an error message,
-                    // the error was handled, and the geometry was replaced with a sphere
-                    // but still good to tell the user about it.
-                    if (errorMessage) {
-                        this.onError(
-                            new FrontEndError(errorMessage, ErrorLevel.WARNING)
-                        );
-                        this.logger.info(errorMessage);
-                    }
-                })
-                .catch((reason) => {
-                    this.onError(new FrontEndError(reason));
-                    this.logger.info(reason);
-                });
```

Comment:
I removed this because `mapKeyToGeom` doesn't actually error. It performs one fallible operation, but converts it in a `catch` to a return type with an error message field. (`mapKeyToGeom` is also changed by this PR, so you can check me on this at the very bottom of the `GeometryStore` diff if you'd like.)

Reply:
LGTM!